JuPnP NPE - Cannot invoke "Object.toString()" because the return value of "org.jupnp.model.state.StateVariableValue.getValue()" is null

I updated to snapshot 3838 this morning and now I'm getting the following flooding out my logs.  Nothing on my network has changed recently.  I was on 3820 (I believe) prior to this and I did not have this issue.  I cleared cache and that had no effect.

openhab> bundle:list -s | grep pnp
246 │ Active │  80 │ 2.7.1.OH1              │ org.jupnp
247 │ Active │  80 │ 4.2.0.202401130308     │ org.openhab.core.config.discovery.addon.upnp
290 │ Active │  80 │ 4.2.0.202401130310     │ org.openhab.core.config.discovery.upnp
291 │ Active │  80 │ 4.2.0.202401130303     │ org.openhab.core.io.transport.upnp


```
2024-01-14 10:05:04.257 [ERROR] [port.upnp.internal.UpnpIOServiceImpl] - Participant threw an exception onValueReceived
java.lang.NullPointerException: Cannot invoke "Object.toString()" because the return value of "org.jupnp.model.state.StateVariableValue.getValue()" is null
        at org.openhab.core.io.transport.upnp.internal.UpnpIOServiceImpl$UpnpSubscriptionCallback.eventReceived(UpnpIOServiceImpl.java:160) [bundleFile:?]
        at org.jupnp.controlpoint.SubscriptionCallback$2.eventReceived(SubscriptionCallback.java:222) [bundleFile:?]
        at org.jupnp.model.gena.RemoteGENASubscription.receive(RemoteGENASubscription.java:114) [bundleFile:?]
        at org.jupnp.protocol.sync.ReceivingEvent$2.run(ReceivingEvent.java:130) [bundleFile:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
        at java.lang.Thread.run(Thread.java:840) [?:?]

```

--- a/bundles/org.openhab.core.io.transport.upnp/src/main/java/org/openhab/core/io/transport/upnp/internal/UpnpIOServiceImpl.java
+++ b/bundles/org.openhab.core.io.transport.upnp/src/main/java/org/openhab/core/io/transport/upnp/internal/UpnpIOServiceImpl.java
@@ -156,11 +156,13 @@ public class UpnpIOServiceImpl implements UpnpIOService, RegistryListener {
             for (UpnpIOParticipant participant : participants) {
                 if (Objects.equals(getDevice(participant), deviceRoot)) {
                     for (Entry<String, StateVariableValue> entry : values.entrySet()) {
-                        try {
-                            participant.onValueReceived(entry.getKey(), entry.getValue().getValue().toString(),
-                                    serviceId);
-                        } catch (Exception e) {
-                            logger.error("Participant threw an exception onValueReceived", e);
+                        Object value = entry.getValue().getValue();
+                        if (value != null) {
+                            try {
+                                participant.onValueReceived(entry.getKey(), value.toString(), serviceId);
+                            } catch (Exception e) {
+                                logger.error("Participant threw an exception onValueReceived", e);
+                            }
                         }
                     }
                     break;

